katello/katello-backup

```diff
@@ -54,49 +95,81 @@ def run_cmd(command, exit_codes=[0])
   end
 end
 
-def backup_db_online
-  puts "Backing up postgres online schema... "
-  run_cmd("runuser - postgres -c 'pg_dumpall -g > #{@dir}/pg_globals.dump'")
-  run_cmd("runuser - postgres -c 'pg_dump -Fc foreman > #{@dir}/foreman.dump'")
-  run_cmd("runuser - postgres -c 'pg_dump -Fc candlepin > #{@dir}/candlepin.dump'")
-  puts "Done."
-
-  puts "Backing up mongo online schema... "
-  run_cmd('mongodump --host localhost --out mongo_dump')
-  puts "Done."
+def snapshot_backup
+  snapsize = @options[:snapshot_size] || "2G"
+  mountdir = "#{@options[:snapshot_mount]}/rootsnapshot" || "/mnt/snap/rootsnapshot"
+  FileUtils.mkdir_p mountdir
+  run_cmd("katello-service stop #{EXCLUDED}")
+  run_cmd("lvcreate -s -nrootsnapshot -L#{snapsize} /dev/#{@options[:snapshot]}/root")
```

Comment:
I'm pretty new to logical volumes, so I'm sure my knowledge base is missing some information. I was under the impression that to take a snapshot of root, which is where all the files we look to backup are located, it would be in `/dev/<volume group name>/root`. Is this incorrect?